packages/discovery-provider/Dockerfile.prod

```diff
@@ -54,11 +54,11 @@ RUN curl -O 'http://openresty.org/package/admin@openresty.com-5ea678a6.rsa.pub'
 RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.10/main' >> /etc/apk/repositories && \
   apk update && \
   apk add \
-  libpq=15.5-r0 \
-  postgresql15-client=15.5-r0 \
-  postgresql15-contrib=15.5-r0 \
-  postgresql15-dev=15.5-r0 \
-  postgresql15=15.5-r0
+  libpq=15.6-r0 \
+  postgresql15-client=15.6-r0 \
+  postgresql15-contrib=15.6-r0 \
+  postgresql15-dev=15.6-r0 \
+  postgresql15=15.6-r0
```

Comment:
I think we just want `postgresql15-client=15.6-r0` and maybe the dev one here. This is for psql right?

Reply:
my gut tells me we should keep all the postgres deps in lockstep with each other, any reason why we would hold back the other packages?

Reply:
resolved to remove in a diff PR per offline discussion